tests/platform_tests/test_environment.py

```diff
@@ -105,6 +106,37 @@ def test_app_meta(self):
             self.raw_app_metadata["configuration"],
         )
 
+    def test_missing_include(self):
```

Comment:
Given what I've mentioned above for the refactoring of the includes computation code to `includes.get_includes` method, you can probably move these tests to `test_includes.py`